javascript/BodyPix/BodyPix_Image/index.html

```diff
@@ -2,18 +2,16 @@
 
 <head>
   <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width">
   <title>BodyPix with Webcam</title>
   <script src="https://unpkg.com/ml5@0.4.0/dist/ml5.min.js" type="text/javascript"></script>
-
-  <style></style>
+  <script src="sketch.js"></script>
```

Comment:
Thanks for the reply!
That sounds about right, I'll have time to hack on it later today 🤘

Reply:
Thanks a bunch! Let me know whenever you're ready and I can review. If you're able to resolve any of the merge conflicts that would be great, otherwise I can also take care :) 

Many thanks!